fe/fe-core/src/main/java/com/starrocks/persist/Storage.java

```diff
@@ -48,27 +50,27 @@ public class Storage {
     public static final String TOKEN = "token";
     public static final String FRONTEND_ROLE = "role";
     public static final String NODE_NAME = "name";
+    public static final String HOST_TYPE = "hostType";
     public static final String IMAGE = "image";
     public static final String IMAGE_NEW = "image.ckpt";
     public static final String VERSION_FILE = "VERSION";
     public static final String ROLE_FILE = "ROLE";
 
-    private int clusterID = 0;
-    private String token;
-    private FrontendNodeType role = FrontendNodeType.UNKNOWN;
-    private String nodeName;
+    private Map<String, String> roleFileMap = new HashMap<>();
```

Comment:
Better not use map to store the deserialized property, the deserialized property should be stored to a definitive var, like toke, nodeName.